deploy/crds/logging.openshift.io_clusterloggings_crd.yaml

```diff
@@ -47,13 +47,21 @@ spec:
                           properties:
                             limits:
                               additionalProperties:
-                                type: string
+                                anyOf:
+                                - type: integer
+                                - type: string
+                                pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
```

Comment:
This is because i upgraded to the fresh from oven `operator-sdk version 0.17.0`. I was facing issues with generated code, so upgraded to latest in an attempt to solve the issue.

Reply:
When I merge #427 we'll stop generating that, only the forwarder will be generated (we can sort out the rest later)  Take the Makefile diff from that PR and the problem will go away.